tests/NuGet.Services.AzureSearch.Tests/IndexBuilderFacts.cs

```diff
@@ -148,7 +151,7 @@ public BaseFacts(ITestOutputHelper output)
 
                 _options
                     .Setup(x => x.Value)
-                    .Returns(() => _config);
+                    .Returns(_config);
```

Comment:
nit I typically us `() => _config` as a pattern so that if a test wants to replace an instance it's easier.

Reply:
(It's `readonly` right now but doesn't really have to if it makes test setup easier)

Reply:
Ah interesting. That makes sense, I'll revert this